base_template.json

```diff
@@ -0,0 +1,148 @@
+                "permission": [
+                    {
+                        "Team": "teamA",
+                        "acl": "admin"
+                    },
+                    {
+                        "Team": "teamB",
+                        "acl": "view"
+                    }
+                ],
+                "Folder": [
+                    {
+                        "name": "innerfolderB",
+                        "Folder": []
+                    }
+                ],
+                "File": [
+                    {
+                        "name": "entityname",
+                        "filepath": "/test/my/path"
+                    }
+                ]
+            }
+        ],
+        "Forum": [
+            {
+                "Thread": {
+                    "title": "Example thread",
+                    "messageMarkdown": "#my markdown",
+                    "Reply": [
+                        {
+                            "messageMarkdown": "response"
+                        }
+                    ]
+                }
+            }
+        ],
+        "Challenge": {
+            "Team": "teamA"
+        },
+        "Table": [
+            {
+                "name": "tableA",
+                "description": "user description of schema",
+                "annotations": {
+                    "key": "pair"
+                },
+                "Column": [
+                    {
+                        "name": "column1",
+                        "defaultValue": "temp",
+                        "maximumSize": 1000,
+                        "columnType": "STRING",
+                        "facetType": "enumeration"
+                    },
+                    {
+                        "name": "column2",
+                        "defaultValue": 2,
+                        "columnType": "INTEGER",
+                        "facetType": "range"
+                    }
+                ]
+            }
+        ],
+        "annotations": {
+            "key": "pair"
+        },
+        "Evaluation": [
+            {
+                "name": "test queue",
+                "description": "This is a testing queue",
+                "submissionInstructionsMessage": "Instrucitons here",
+                "submissionReceiptMessage": "Successful submission!",
+                "quota": {
+                    "firstRoundStart": 11111,
+                    "roundDurationMillis": 1,
+                    "numberOfRounds": 2,
+                    "submissionLimit": 5
+                },
+                "permission": [
+                    {
+                        "Team": "teamA",
+                        "acl": "admin"
+                    },
+                    {
+                        "Team": "teamB",
+                        "acl": "view"
+                    }
+                ]
+            }
+        ],
+        "Wiki": {
+            "_comment": "Use one of these below keys",
+            "WikiTemplate": "syn18058986",
```

Comment:
This should just use the `oneof` in the json schema which allows either a `syn` string or the `wikistructure`